src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: left 19px top 32px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__title {
+    margin-top: 206px;
+    margin-bottom: 4px;
+    font-family: $main-font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-text-color;
+    padding-inline: $default-padding-inline;
+  }
+
+  &__product-code {
+    font-family: $main-font-family;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-text-color;
+    margin: inherit;
+    margin-bottom: 16px;
```

Comment:
Using 'margin: inherit;' might not work as expected if the parent element doesn't have a defined margin. Consider specifying a concrete value or removing it if not necessary.